yb-voyager/cmd/callhome_payload_builder.go

```diff
@@ -471,7 +472,14 @@ func buildCallhomeSchemaOptimizationChanges() []callhome.SchemaOptimizationChang
 		schemaOptimizationChanges = append(schemaOptimizationChanges, callhome.SchemaOptimizationChange{
 			OptimizationType: PK_HASH_SPLITTING_CHANGE_TYPE,
 			IsApplied:        schemaOptimizationReport.PKHashSplittingChange.IsApplied,
-			Objects:          []string{},
+			Objects:          getAnonymizedConstraintNamesFromConstraints(schemaOptimizationReport.PKHashSplittingChange.ModifiedConstraints),
```

Comment:
(For both pk hash, pk timestamp range)

Both from a report perspective as well as call-home, I think "tables" make more sense as compared to "constraints"

Can be taken up later since this might be a big change.

Reply:
Hmm I think that makes sense, I will try to make that change if possible else will do later